src/MaxText/pyconfig_deprecated.py

```diff
@@ -442,10 +442,8 @@ def validate_qwen3_next_config(keys: dict):
     keys: the raw config in dict form
 
   """
-  if keys["sparse_matmul"]:
-    raise ValueError(
-        "For Qwen3-Next, sparse_matmul must be False for now. The dense path has been verified against reference."
-    )
+  if int(keys["gdn_num_value_heads"]) % int(keys["gdn_num_key_heads"]) != 0:
```

Comment:
I think this file has been deprecated and src/MaxText/configs/types.py is the new place for checks